rca/shortcourses/models.py

```diff
@@ -240,12 +242,21 @@ def _format_booking_bar(self, register_interest_link, access_planit_data):
         # a modal, this link is also used as a generic interest link too though.
         booking_bar["link"] = register_interest_link
 
+        # Swap to 'apply' if an aplication form has been set
+
         if access_planit_data:
+            booking_action = "Book"
+            if self.application_form_url:
+                booking_action = "Apply"
```

Comment:
Could reduce to `booking_action = "Apply" if self.application_form_url else "Book"`